acceptance/python/Dockerfile

```diff
@@ -5,5 +5,5 @@ RUN pip install --upgrade \
 	pycco \
 	websocket-client~=0.47.0 \
 	pytest~=3.0 \
-	bigchaindb-driver==0.5.2 \
+	bigchaindb-driver==0.5.3 \
```

Comment:
If we use the `~` operator we can let the package manager handle minor versions. What about `bigchaindb-driver~=0.5.0`?

Reply:
Good idea.